content/en/blog/basics-for-passwordless/index.md

```diff
@@ -0,0 +1,136 @@
+or even replacing passwords altogether, might initially seem overwhelming.
+
+## FIDO2 and WebAuthn Standards Ease the Job
+
+Integrating the authenticators into your web application happens through common standards
+created by [the FIDO alliance](https://fidoalliance.org/). It is good news for us
+application developers: we don't have to care about the dirty details of each
+authenticator implementation. Our job is to take into use the platform or browser APIs
+that enable the creation and use of the authenticator-managed credential.
+The operating system or browser handles needed user interaction and shows standard dialogs when appropriate.
+
+{{< imgproc cover Fit "825x825" >}}
+<em>The team implements passwordless support by integrating credential handling into
+the client application and the backend authentication service. Client applications
+can use the platform WebAuthn capabilities through the native APIs or browser JavaScript implementation.
+The backend service must support at least credential registration and credential-based user authentication.
+</em>
+{{< /imgproc >}}
+
+Furthermore, we need to have a backend service in place that is capable of storing and verifying
+these credentials so that the authentication can take place correctly. The service must support
+[the W3C WebAuthn standard](https://www.w3.org/TR/webauthn/) so that the backend
+API responses are compatible with the client-side authenticator logic.
+
+## To Buy or To Build?
+
+Choosing one of many authentication service providers may be the most straightforward path on your
+passwordless journey. A service provider typically gives you access to a backend as
+a SaaS or a product you host yourself. The backend has the above capabilities to store and verify
+your users' public keys. In addition to backend functionality, the service providers offer
+client libraries that enable you to add matching authenticator support to your application,
+whether a web or a native application.
+
+{{< imgproc platform Fit "825x825" >}}
+<em>Platform and browser handle the user interaction dialogs when using authenticators.</em>
+{{< /imgproc >}}
+
+Another option is to implement the WebAuthn support yourself. Some excellent [open-source libraries](https://github.com/herrjemand/awesome-webauthn)
+already exist that ease the development of your backend service's functionality for
+public key handling. [Browser support for WebAuthn capabilities](https://caniuse.com/webauthn) is
+rather good, and the integration to the web application is straightforward
+once the backend is in place. One can utilize dedicated
+client libraries for native applications (for example, [iOS](https://developer.apple.com/documentation/authenticationservices/public-private_key_authentication)
+and [Android](https://developer.android.com/training/sign-in/passkeys)).
+
+## Example of Getting Hands Dirty
+
+In our OSS decentralized identity agency project, we implemented WebAuthn logic ourselves.
```

Comment:
We could reference [this section of our earlier blog post](https://findy-network.github.io/blog/2021/11/09/anchoring-chains-of-trust/#headless-fido2webauthn-authenticator).